Incomplete source tarball for 1.2.0 RC5 -> build error in src/3rdparty/qt5-x11embed

Trying to package lmms 1.2.0 RC5 for Rosa Linux Fresh R10 (a Mandriva derivative) on x86_64.
Cmake stops with the error message:
************************************************
CMake Error at src/3rdparty/CMakeLists.txt:6 (add_subdirectory):
The source directory /tmp/abf/rpmbuild/BUILD/lmms-1.2.0-rc5/src/3rdparty/qt5-x11embed
does not contain a CMakeLists.txt file.
************************************************
Attached the full log
[BuildFailure.txt](https://github.com/LMMS/lmms/files/1612749/BuildFailure.txt)

Indeed the directory src/3rdparty/qt5-x11embed is empty in the RC5 tarball, while it is absent in the RC4 one (and using it produces a successful build).
So I guess you will need to regenerate the source tarball for 1.2.0 RC5...

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,12 +15,12 @@ matrix:
           osx_image: xcode8.2
         - env: QT5=
         - env: QT5=True
-        - env: QT5=True TARGET_OS=win32
-        - env: QT5=True TARGET_OS=win64
+        - env: QT5=True TARGET_OS=win32 TARGET_DEPLOY=True
+        - env: QT5=True TARGET_OS=win64 TARGET_DEPLOY=True
         - os: osx
           osx_image: xcode8.2
           env: QT5=True
-        - env: TARGET_OS=debian-sid
+        - env: TARGET_OS=debian-sid TARGET_DEPLOY=True
         - env: TARGET_OS=debian-sid TARGET_ARCH=i386
         - compiler: clang
           env: TARGET_OS=debian-sid
@@ -41,12 +41,15 @@ deploy:
     provider: releases
     api_key:
         secure: d4a+x4Gugpss7JK2DcHjyBZDmEFFh4iVfKDfITSD50T6Mc6At4LMgojvEu+6qT6IyOY2vm3UVT6fhyeuWDTRDwW9tfFlaHVA0h8aTRD+eAXOA7pQ8rEMwQO3+WCKuKTfEqUkpL4wxhww8dpkv54tqeIs0S4TBqz9tk8UhzU7XbE=
-    file: lmms-${TRAVIS_TAG:1}-$TARGET_OS.exe
+    file_glob: true
+    file:
+        - lmms-${TRAVIS_TAG:1}-$TARGET_OS.exe
+        - /var/cache/pbuilder/result/lmms_*.tar.xz
     skip_cleanup: true
     on:
         tags: true
         all_branches: true
-        condition: '("$TARGET_OS" = win??) && "$QT5"'
+        condition: '"$TARGET_DEPLOY" = True'
     repo: LMMS/lmms
 notifications:
     webhooks:

